proto/pocket/migration/tx.proto

```diff
@@ -176,6 +193,8 @@ message MsgClaimMorseApplicationResponse {
 
   // The application which was staked as a result of the claim.
   application.Application application = 5 [(gogoproto.jsontag) = "application"];
+
+  // Next free index: 6
```

Comment:
I'm not a big fan of out of order indexes and relying on comments to identify the next one. I believe it's a bad dx.

We're already having them out of order before even having Morse claiming records onchain.

I'm in favor of fully opting for proto migration if it's necessary.

Reply:
Can we move this discussion to an issue or something? I don't think it should block this PR.

Reply:
That's what we should do.

Marking the issue as resolved.